### ISSUE TITLE
Add support for pix2pix and inpainting models

### ISSUE THREAD

**Issue by ljleb** [ljleb/sd-mecha#5]
Merging a pix2pix or inpainting model with a normal model yet does not work. We need to catch dimension mismatches in the shapes of the tensors and take appropriate action. I need to look around on github, I'm pretty sure there is code for this already somewhere.

**Comment by Enferlain** [comment]
This?

https://github.com/vladmandic/automatic/blob/3c952675fefd2c94b817940ffbd4cd94fd5876c9/modules/merging/merge.py#L335

**Comment by ljleb** [comment]
Yes, but I'm not sure this is how it's supposed to be done. Instead of ignoring the keys, we should test slicing, merging and concat.
